app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatabaseService.kt

```diff
@@ -173,12 +173,12 @@ class FirestoreDatabaseService internal constructor(
         try {
             ref.delete().await()
         } catch (e: FirebaseFirestoreException) {
-            Log.w(this.toString(), "Removing picture ${picture.url} from ${this.db} failed", e)
+            Log.w(this.toString(), "Removing picture ${picture.id} from ${this.db} failed", e)
         }
         try {
-            storage.getReferenceFromUrl(picture.url).delete().await()
+            storage.getReferenceFromUrl(picture.picture.toString()).delete().await()
         } catch (e: FirebaseException) {
-            Log.w(this.toString(), "Removing the image ${picture.url} from storage failed", e)
+            Log.w(this.toString(), "Removing the image ${picture.id} from storage failed", e)
```

Comment:
I don't agree with this change, the Firebase Storage does not store the id, hence it does not provide any information on which picture failed to be deleted if the picture already got deleted from the Firestore database.